server/repository/src/db_diesel/rnr_form_line_row.rs

```diff
@@ -38,10 +41,13 @@ table! {
 }
 
 joinable!(rnr_form_line -> rnr_form (rnr_form_id));
-joinable!(rnr_form_line -> item (item_id));
+joinable!(rnr_form_line -> item_link (item_id));
```

Comment:
How bad behaviour is it to have the field as `item_id` on rnr_form_line instead of `item_link_id`? 🙈